ursh/indico_ursh/plugin.py

```diff
@@ -59,4 +59,4 @@ def get_blueprints(self):
 
     def _inject_ursh_button(self, target, **kwargs):
         if self.settings.get('api_key') and self.settings.get('api_host'):
-            return render_plugin_template('ursh_button.html', target=target)
+            return render_plugin_template('ursh_button.html', target=target, **kwargs)
```

Comment:
`**kwargs` of a signal handler shouldn't be used; it's just for forward compatibility with new arguments added to a signal. Any param you expect to handle should be defined explicitly - this is better here anyway since you can now give it a default value ;)